source/documentation/team-dashboard.md

```diff
@@ -0,0 +1,72 @@
+# Team Dashboard
+
+We now have dashboards!
+
+We currently display dashboards on our TV monitors (two of them), running on ChromeOS. This section 
+outlines what's currently displayed on them.
+
+## Technical
+
+The top monitor has technical dashboards worth keeping an eye on throughout the day.
+
+### Grafana
+
+Grafana is an open source analytics and monitoring platform, and we use it for monitoring the health 
+of GovWifi in realtime.
+
+In the past, GovWifi used to be part of a shared organisation on Grafana. Now, GovWifi has its own
+organisation, and we have admin-level access so we can now make our own changes without having to 
+always involve RE. 
+
+We currently have two dashboards in Grafana:
+
+1. The first is for monitoring performance of GovWifi Product Pages and Tech Docs, and is available
+[here](https://grafana-paas.cloudapps.digital/d/KMxSG3DWk/govwifi?orgId=1&refresh=10s).
```

Comment:
```suggestion
[here](https://grafana-paas.cloudapps.digital/d/KMxSG3DWk/govwifi).
```
Removing org and refresh, since they are not needed (they are filled by the defaults) and the org is still the shared one, whereas we have moved to our own org